packages/radio-group/src/Radio.tsx

```diff
@@ -314,20 +360,43 @@ function Radio({
           focusTargetElement={inputElement}
           className={cx(radioBoxStyle, radioBoxSize[normalizedSize])}
           borderRadius="100%"
+          color={!darkMode ? { focused: 'transparent' } : undefined} // TODO: Refresh - overriding the focus style for light mode
           {...inputDisplayWrapper.prop}
         >
           <div
             {...inputDisplay.prop}
-            className={cx(divStyle, divColorSet[mode], divSize[normalizedSize])}
+            className={cx(
+              divStyle,
+              divColorSet(mode, normalizedSize),
+              divSize[normalizedSize][mode],
+              {
+                [css`
+                  &:before {
+                    top: -5px;
+                    bottom: -5px;
+                    left: -5px;
+                    right: -5px;
+                  }
+
+                  &:after {
+                    box-shadow: inset 0 -1px 0 0 #f1f1f1,
+                      0 1px 0 0 rgba(0, 0, 0, 0.08),
+                      0 1px 1px 0 rgba(6, 22, 33, 0.22);
+                  }
+                `]: darkMode,
+              },
+            )}
           />
         </InteractionRing>
 
         <div
           className={cx(
             css`
-              margin-left: ${size === Size.XSmall ? 4 : 8}px;
+              margin-left: ${size === Size.XSmall && darkMode === true
+                ? 4
+                : 8}px;
```

Comment:
I'd probably add the default margin to the base styles object and then override it using `cx` conditionals